test/unit_tests/apps/teachers-digital-platform/js/expandable-facets-spec.js

```diff
@@ -66,9 +66,9 @@ describe( 'Expandable facets', () => {
     ef = document.querySelector( '.o-expandable-facets' );
     expandableFacet = new ExpandableFacets( ef );
     expandableFacet.init();
-    efLabel = document.querySelector( '.o-expandable-facets_checkbox ~ .a-label' );
+    // efLabel = document.querySelector( '.o-expandable-facets_checkbox ~ .a-label' );
     efTarget = document.querySelector( '.o-expandable-facets_target' );
-    efHeader = document.querySelector( '.o-expandable_header' );
+    // efHeader = document.querySelector( '.o-expandable_header' );
```

Comment:
Commented out because they are unused? Coverage for this file is at 94.44%. We could look if these are worth including in additional tests, or they could be deleted.

Reply:
I made an attempt to check if label or header are hidden.
@anselmbradford What is the command line to run unit_tests that would generate code coverage?